app/src/main/java/io/intrepid/contest/screens/adminstatus/ConfirmStartContestDialog.java

```diff
@@ -0,0 +1,49 @@
+package io.intrepid.contest.screens.adminstatus;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.support.v4.app.FragmentManager;
+import android.widget.Button;
+
+import io.intrepid.contest.R;
+
+public class ConfirmStartContestDialog extends DialogFragment {
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        AlertDialog dialog = (AlertDialog) getDialog();
+        if (dialog != null) {
+            Button positiveButton = dialog.getButton(Dialog.BUTTON_POSITIVE);
+            Button negativeButton = dialog.getButton(Dialog.BUTTON_NEGATIVE);
+
+            int textColor = getContext().getResources().getColor(R.color.colorPrimary);
+            positiveButton.setTextColor(textColor);
+            negativeButton.setTextColor(textColor);
+            positiveButton.setAllCaps(false);
+            negativeButton.setAllCaps(false);
+        }
+    }
+
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        final Context ctx = getActivity();
+        return new AlertDialog.Builder(ctx)
+                .setTitle(R.string.not_all_contestants_received_prompt)
+                .setMessage(R.string.start_contest_prompt)
+                .setPositiveButton(R.string.common_yes, (dialog, which) -> {
+
```

Comment:
If we're leaving these blank it would be a good place for a "TODO", we can also add some timber statements for debugging.